views/partials/footer.html

```diff
@@ -10,8 +10,15 @@
 
     <br>
 
-    © Science Stories 2018 <br>
+    © Science Stories <span id="year-stamp"></span> <br>
     <a href="https://www.seals-nutt.com">Developed By Seals-Nutt</a><br>
     <a href="/sitemap">sitemap</a>
   </p>
 </section>
+
+<script>
+  let date = new Date();
```

Comment:
nit: since these values aren't changing, consider `const` over `let`. Similarly, it can be condensed down a bit:
```
const yearStamp = document.getElementById('year-stamp');
yearStamp.innerHTML = new Date().getFullYear();
```